src/server.test.ts

```diff
@@ -286,3 +286,28 @@ test('Lists entries', async () => {
 
   expect(parachutesSongs2.directories).toEqual([])
 })
+
+test('Works with a deploy-scoped store', async () => {
```

Comment:
I have committed this test first and you can see it failing: https://github.com/netlify/blobs/pull/119/commits/d7468982e9dce4a36baec8b523d173f68cfefb4d